### PR TITLE
Create dependabot.yml

### PR DESCRIPTION
**Pull request by abdulsattar**
## Does this PR introduce breaking changes?

* ✅ `No, it does not introduce breaking changes.`
* 🚨 `Yes, it does introduce breaking changes.`

If yes, please describe the impact and migration path for existing applications.

## The PR fulfills these requirements:
* Have tests for the proposed changes been added? ✅ / ❌
* Have you followed [these instructions](../CONTRIBUTING.md#-commit-message-conventions) to clearly describe the issue being fixed or feature enhanced? ✅ / ❌

### CODE REVIEW

File: .github/dependabot.yml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# To get started with Dependabot version updates, you'll need to specify which
+# package ecosystems to update and where the package manifests are located.
+# Please see the documentation for all configuration options:
+# https://help.github.com/github/administering-a-repository/configuration-options-for-dependency-updates
+
+version: 2
+updates:
+  - package-ecosystem: "npm" # See documentation for possible values
```
Comment (ekashida) [inline .github/dependabot.yml:8]:
```suggestion
  - package-ecosystem: "npm" # yarn-managed projects should also be configured as "npm"
```

File: .github/dependabot.yml [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+    directory: "/" # Location of package manifests
```
Comment (ekashida) [inline .github/dependabot.yml:9]:
```suggestion
    directory: "/" # https://dependabot.com/blog/lerna-support/
```

**Review by ekashida — COMMENTED** [review]
Since Monday 5:00 UTC is Monday 21:00 PST, it might be better to specify `schedule.day = "sunday"` to ensure we see any updates before the weekly dependency update.

**Comment by pmdartus** [comment]
Will this produce a single PR for all the dependency update? Or will it create one PR per dependency?

**Comment by abdulsattar** [comment]
> Will this produce a single PR for all the dependency update? Or will it create one PR per dependency?

Looks like a single PR: https://github.com/dependabot/dependabot-core/issues/1190

**Review by pmdartus — APPROVED** [review]

**Comment by pmdartus** [comment]
After discussing with @abdulsattar, we will keep updating the dependencies by hand for now. A single PR per dependency will create a lot of noise on this repo.

**Pull request closed without merging**